src/components/shared/WithSize.js

```diff
@@ -37,17 +37,34 @@ export function withSize<
 > {
   return class WithSizeWrapper extends React.PureComponent<*, State> {
     _resizeListener: () => void;
+    _visibilitychangeListener: () => void;
```

Comment:
The project uses `camelCasing` for variable and function names, so to be consistent, it should be `_visibilityChangeListener` with the `C` capitalized.